services/app-api/src/resolvers/APIKey/createAPIKey.test.ts

```diff
@@ -7,7 +7,7 @@ describe('createAPIKey', () => {
     it('creates a new API key', async () => {
         const jwt = newJWTLib({
             issuer: 'mctestiss',
-            signingKey: 'foo',
+            signingKey: Buffer.from('123af', 'hex'),
```

Comment:
Can you add a comment here for why `123af` is used here?

Reply:
I wanted a short dumb key for testing, and since we're passing in keys as hex encoded strings I wanted to do that the same way we are elsewhere too